src/pages/agent/tests-to-run/tests-to-run.tsx

```diff
@@ -57,6 +124,24 @@ export const TestsToRun = ({ agentType = "Agent" }: Props) => {
     .reduce((test, testType) => ({ ...test, [testType.type]: testType }), {}) as TestsInfo;
   const previousBuildAutoTestsCount = AUTO?.summary?.testCount || 0;
 
+  const stub = useMemo(() => (testsToRun.length > 0
+    ? (
+      <Stub
+        icon={<Icons.Test height={104} width={107} />}
+        title="No results found"
+        message="Try adjusting your search or filter to find what you are looking for."
+      />
+    )
+    : (
+      <Stub
+        icon={<Icons.Test height={104} width={107} />}
+        title={status === AGENT_STATUS.BUSY ? "Build tests are loading" : "No tests available yet"}
+        message={status === AGENT_STATUS.BUSY
+          ? "It may take a few seconds."
+          : "Information about project tests will appear after the first launch of tests."}
```

Comment:
Remove the ternary operator because there is only one state of the stub.
The text should be: **There is no information about the suggested to run tests
in this build.**